packages/organizations/origin-organization-irec-api/src/beneficiary/dto/beneficiary.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { plainToClass, Type } from 'class-transformer';
+import { PublicOrganizationInfoDTO } from '@energyweb/origin-backend';
+
+export class BeneficiaryDTO {
+    @ApiProperty({ type: Number })
+    id: number;
+
+    @ApiProperty({ type: Number })
+    irecBeneficiaryId: number;
+    @ApiProperty({ type: PublicOrganizationInfoDTO })
+    @Type(() => PublicOrganizationInfoDTO)
+    organization: PublicOrganizationInfoDTO;
+
+    @ApiProperty({ type: Number })
+    ownerOrganizationId: number;
```

Comment:
If this is mean to be the organization that owns the beneficiary record, I suggest renaming this field to `ownerId` as it's used in other parts of the system.

Reply:
changed